docs/keycloak.rst

```diff
@@ -0,0 +1,238 @@
+Keycloak Integration with ZOO-Project
+=====================================
+
+This guide documents the integration of Keycloak into the ZOO-Project using a remote or Docker-hosted Keycloak instance. The goal is to secure OGC API endpoints using HTTP Bearer Token authentication.
+
+Overview
+--------
+
+ZOO-Project's endpoints like `/ogc-api/jobs`, `/ogc-api/processes`, `/ogc-api/stac`, and `/ogc-api/raster` can be protected using Keycloak-based authentication via the `libapache2-mod-auth-openidc` Apache module.
+
+Two Keycloak clients are required:
+
+1. **API Client** (for Apache to validate Bearer tokens)
+2. **Web Client** (for the Nuxt web app to authenticate users and obtain tokens)
+
+Docker Setup
+------------
+
+### docker-compose-ko.yml (Keycloak service)
+
+This file is located at: `zoo-project/docker/nuxt-client/`
```

Comment:
This directory does not exist in the repository. 

Please add this directory and the required files to your pull request.